text/0000-on-modifier.md

```diff
@@ -0,0 +1,193 @@
+- Start Date: 2019-03-21
+- Relevant Team(s): Ember.js, Learning
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# `{{on}}` Modifier
+
+## Summary
+
+Add an `{{on}}` modifier for adding event listeners to elements.
+
+## Motivation
+
+Currently, there are two ways to bind event listeners to elements in Ember
+templates with built-in, official Ember APIs:
+
+1. Use the `{{action}}` element modifier
+2. Use the `on*=` property bindings
+
+Both of these solutions are problematic for a number of reasons.
+
+The `{{action}}` modifier:
+
+- Uses a non-standard AST transform to pass `this` in order to bind the context
+  of the template (the component) to the function (typically a component
+  method). This is problematic behavior, not just because of the transform, but
+  because of the implicit state that is being passed to the modifier in the
+  first place, and the fact that the modifier is "magical" in the sense that it
+  can do things that no other modifier or helper can do.
+- Binds to the `click` event by default, which requires a bit of knowledge to
+  understand on first glance, and requires a bit of extra knowledge to figure
+  out how to bind to other events. Compared to `onclick=`, this is less
+  explicit, and requires more knowledge of the framework out of the box.
+- Is confusing when used with the `@action` decorator to bind the function, and
```

Comment:
This is a problem the `@action` decorator should be responsible for handling, not `action` that has been there since (almost) forever.